modules/xfem/test/tests/moving_interface/verification/1D_rz_homog1mat.i

```diff
@@ -0,0 +1,187 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+# XFEM Moving Interface Verification Problem
+# Dimensionality:                                   quasi-1D
+# Coordinate System:                                      rz
+# Material Numbers/Types:   homogeneous 1 material, 2 region
+# Element Order:                                         1st
+# Interface Characteristics: u independent, prescribed level set function
+# Description:
+#   A simple transient heat transfer problem in cylindrical coordinates designed
+#   with the Method of Manufactured Solutions. This problem was developed to
+#   verify XFEM performance in the presence of a moving interface for linear
+#   element models that can be exactly evaluated by FEM/Moose. Both the
+#   temperature solution and level set function are designed to be linear to
+#   attempt to minimize error between the Moose/exact solution and XFEM results.
+#   Thermal conductivity is a single, constant value at all points in the system.
+# Results:
+#   The temperature at the left boundary (x=1) exhibits the largest difference
+#   between the FEM/Moose solution and XFEM results. We present the XFEM results
+#   at this location with 10 digits of precision:
+#     Time    Expected Temperature    XFEM Calculated Temperature
+#      0.2                  440         440
+#      0.4                  480         480.0008118
+#      0.6                  520         520.0038529
+#      0.8                  560         560.0089177
+#      1.0                  600         600.0133344
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+
+[GlobalParams]
+  order = FIRST
+  family = LAGRANGE
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 4
+  ny = 1
+  xmin = 1.0
+  xmax = 2.0
+  ymin = 0.0
+  ymax = 0.5
+  elem_type = QUAD4
+[]
+
+[XFEM]
+  qrule = moment_fitting
+  output_cut_plane = true
+[]
+
+[UserObjects]
+  [./level_set_cut_uo]
+    type = LevelSetCutUserObject
+    level_set_var = ls
+    heal_always = true
+  [../]
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[AuxVariables]
+  [./ls]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./heat_cond]
+    type = MatDiffusion
+    variable = u
+    D_name = 'diffusion_coefficient'
+  [../]
+  [./vol_heat_src]
+    type = BodyForce
+    variable = u
+    function = src_func
+  [../]
+  [./mat_time_deriv]
+    type = TestMatTimeDerivative
+    variable = u
+    mat_prop_value = rhoCp
+  [../]
+[]
+
+[AuxKernels]
+  [./ls_function]
+    type = FunctionAux
+    variable = ls
+    function = ls_func
+  [../]
+[]
+
+[Constraints]
+  [./xfem_constraint]
+    type = XFEMSingleVariableConstraint
+    variable = u
+    geometric_cut_userobject = 'level_set_cut_uo'
+    use_penalty = true
+    alpha = 1e5
```

Comment:
Is this penalty enforcement the source of most of your error?  I assume you studied the effect of that on the accuracy.

Reply:
The alpha value was suggested by Wen as something to start with. Decreasing the value of alpha does have a small but appreciable effect on the error, but increasing the value from here has a fairly small impact on the resulting temperature. For example, looking at the final timestep value of u at 0 on this simulation, we have u=600.235 (alpha=1e3), u=600.013 (alpha=1e5), u=600.011 (alpha=1e7). Anyways, the alpha value of 1e5 seems good to me.